Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -2310,7 +2322,7 @@ namespace Babylon
                         InstanceMethod("trySetFeaturePointCloudEnabled", &XRSession::TrySetFeaturePointCloudEnabled),
                         InstanceMethod("trySetPreferredPlaneDetectorOptions", &XRSession::TrySetPreferredPlaneDetectorOptions),
                         InstanceMethod("trySetMeshDetectorEnabled", &XRSession::TrySetMeshDetectorEnabled),
-                        InstanceMethod("trySetPreferredMeshDetectorOptions", &XRSession::TrySetPreferredMeshDetectorOptions),
+                        InstanceMethod("trySetPreferredMeshDetectorOptions", &XRSession::TrySetPreferredMeshDetectorOptions)
```

Comment:
>) [](start = 123, length = 1)

nit: may want to restore the `,` it's useful to leave a trailing one so that future diffs and merge conflicts don't have issues